src/app.scss

```diff
@@ -4,3 +4,17 @@
   $theme-image-path: $theme-image-path,
   $theme-show-notifications: false
 );
+
+html,
+body {
+  min-height: 100vh;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+}
+
+main {
+  flex: 1;
+}
```

Comment:
this is to push the copyright tag to the bottom of the page. Probably not much of an issue in the final state of pages, but on stubs or pages with small `<main>` sections, the footer is in the middle of the page

Reply:
@LouisFettet's suggestion seems like the better choice just in case `main` isn't set up as expected, assuming the `position: sticky` is widely-supported enough or falls back to an acceptable behavior.

Reply:
yup, that works! I dig it, making that change.